### PR TITLE
Add information for timestamp accuracy

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

File: mountainduck/issues/index.md [hunk 1]
```diff
@@ -263,6 +263,11 @@ By default, files starting with `.` aren't hidden by Windows Explorer. You can c
 
     browser.hidden.regex=\\..*
 
+**Timestamp Accuracy**<br/>
+Windows has a sub-second timestamp accuracy while protocols like SFTP has an accuracy of a second. This discrepancy causes problems while copying files. 
+- Reported for [Robocopy](https://stackoverflow.com/questions/15145659/what-do-i-and-i-in-regex-mean):<br/>
```
Comment (AliveDevil) [inline mountainduck/issues/index.md:268]: This link doesn't refer to anything related to this part.

File: mountainduck/issues/index.md [hunk 2]
```diff
@@ -263,6 +263,11 @@ By default, files starting with `.` aren't hidden by Windows Explorer. You can c
...
+Using the command `robocopy /MIR /FET` instead of `robocopy /MIR` fixes the behavior.
```
Comment (AliveDevil) [inline mountainduck/issues/index.md:269]:
```suggestion
Using the command `robocopy /MIR /FFT` instead of `robocopy /MIR` fixes the behavior.
```

**Review by AliveDevil — CHANGES_REQUESTED** [review]
Please include the link to [microsoft](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) documentation for robocopy.

Review typo in robocopy command.

File: mountainduck/issues/index.md [hunk 3]
```diff
@@ -263,6 +263,11 @@ By default, files starting with `.` aren't hidden by Windows Explorer. You can c
 
     browser.hidden.regex=\\..*
 
+**Timestamp Accuracy**<br/>
```
Comment (dkocher) [inline mountainduck/issues/index.md:266]: Change title to include `Robocopy`

File: mountainduck/issues/index.md [hunk 4]
```diff
@@ -263,6 +263,13 @@ By default, files starting with `.` aren't hidden by Windows Explorer. You can c
 
     browser.hidden.regex=\\..*
 
+**Robocopy Timestamp Accuracy**<br/>
+Windows has a sub-second timestamp accuracy while protocols like SFTP has an accuracy of a second. This discrepancy causes problems while copying files.
```
Comment (AliveDevil) [inline mountainduck/issues/index.md:267]: Requested change by @dkocher:
```suggestion
Windows has a sub-second timestamp accuracy while protocols like SFTP has an accuracy of a second. This discrepancy causes unnecessary transfers when subsequent runs find non-matching timestamps on unchanged files.
```

Fix typo:
```suggestion
Windows has a sub-second timestamp accuracy while protocols like SFTP have an accuracy of a second. This discrepancy causes problems while copying files. 
```

**Review by AliveDevil — CHANGES_REQUESTED** [review]

**Review by AliveDevil — APPROVED** [review]

**Pull request merged**